models/Amity.py

```diff
@@ -0,0 +1,221 @@
+        if target not in self.people["FELLOWS"]:
+            print "{0} does not exit".format(person_name)
+            try:
+                sys.exit()
+            except:
+                print "Exception accomodates tdd"
+        if person_type == "FELLOW":
+            self.deallocate_fellow(target, assigned)
+            new_room = self.get_roomobject(room_name)
+            new_room.occupants.append(target)
+            self.allocated_persons.append([target, new_room])
+            print "{0} was reallocated to {1}".format(
+                target.name, new_room.room_name)
+
+    def get_roomname(self, rooms):
+        all_room_names = []
+        for room in rooms:
+            all_room_names.append(room.room_name)
+        return all_room_names
+
+    def get_fellowobject(self, person_name):
+        for person in self.people["FELLOWS"]:
+            if person.name == person_name:
+                return person
+
+    def get_roomobject(self, room_name):
+        for room in self.rooms["LivingSpace"]:
+            if room.room_name == room_name:
+                return room
+
+    def get_staffobject(self, person_name):
+        for person in self.people["STAFF"]:
+            if person.name == person_name:
+                return person
+
+    def get_listofrooms(self):
+        all_rooms = []
+        for room in self.rooms["LivingSpace"]:
+            if len(room.occupants) < 4:
+                all_rooms.append(room)
+        for rooms in self.rooms["Office"]:
+            if len(room.occupants) < 4:
+                all_rooms.append(rooms)
+        return all_rooms
+
+    def get_livingspaceobject(self, room_name):
+        for room in self.rooms["LivingSpace"]:
+            if room.room_name == room_name:
+                return room
+
+    def return_people_allocated(self):
+        for person in self.people["FELLOWS"]:
+            if person.allocated is True:
+                return person
+
+    def return_room_allocated(self, person_object):
+        for room in self.rooms["LivingSpace"]:
+            if person_object in room.occupants:
+                print room.occupants
+                return room
+            else:
+                return "Person not allocated, check in unallocated lists"
+
+    def print_rooms(self):
+        room_names_list = []
+        rooms = self.get_listofrooms()
+        for room in rooms:
+            room_names_list.append(room.room_name)
+        return room_names_list
+
+    def print_allocations(self):
+        if len(self.allocated_persons) > 0:
+            for person in self.allocated_persons:
+                try:
+                    print "{0} was allocated to {1}".format(
+                        person[0].name, person[1].room_name)
+                except AttributeError:
+                    print "END OF LIST"
+        else:
+            print "Fellows have not been allocated yet, Please add a fellow"
+
+    def print_unallocated(self):
```

Comment:
Not really sure but shouldn't this method print out a list of all unallocated fellows? Similar to printing allocations?